### DISCUSSION TITLE
Unable to upgrade v7

### DISCUSSION THREAD

**Discussion by theolavaux** [pnpm/pnpm#4896] [Q&A] [answered]
I have installed `pnpm@6.32.20` through Corepack but now, I'm unable to version bump to v7 successfully.
The installs seems ok but it's still using the old version.
How could I fix this?

<img width="268" alt="CleanShot 2022-06-17 at 12 35 58@2x" src="https://user-images.githubusercontent.com/25274586/174282105-d4831abf-6c69-42d7-82b0-f81b18e8cebd.png">

**Comment by zkochan** [comment, marked as answer]
I have no idea, if you only used corepack, you should probably ask in the corepack repo https://github.com/nodejs/corepack
